cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -225,18 +292,26 @@ def test_ansible_playbook_button_crud(ansible_catalog_item, appliance, request):
     assert not button.exists
 
 
-@pytest.mark.meta(blockers=[BZ(1548562, forced_streams=["5.9"])])
-def test_embedded_ansible_custom_button(full_template_vm, custom_vm_button, service_request,
-                                        service, appliance):
-    view = navigate_to(full_template_vm, "Details")
+def test_custom_button_order_ansible_playbook_service(full_template_vm_modscope, custom_vm_button,
+        service_request, service, appliance):
+    view = navigate_to(full_template_vm_modscope, "Details")
     view.toolbar.custom_button(custom_vm_button.group.text).item_select(custom_vm_button.text)
     submit_button = WButton(appliance.browser.widgetastic, "Submit")
     submit_button.click()
     wait_for(service_request.exists, num_sec=600)
     service_request.wait_for_request()
     view = navigate_to(service, "Details")
+    hosts = view.provisioning.details.get_text_of("Hosts")
     if custom_vm_button.inventory == "Localhost":
-        assert view.provisioning.details.get_text_of("Hosts") == "localhost"
+        assert hosts == "localhost"
     else:
-        assert view.provisioning.details.get_text_of("Hosts") == full_template_vm.ip_address
+        assert hosts == full_template_vm_modscope.ip_address
     assert view.provisioning.results.get_text_of("Status") == "successful"
+
+
+def test_alert_run_ansible_playbook(full_template_vm_modscope, alert_profile, request, appliance):
+    full_template_vm_modscope.add_tag("Service Level", "Gold")
+    full_template_vm_modscope.remove_tag("Service Level", "Gold")
+    request.addfinalizer(lambda: appliance.ssh_client.run_command(
+        "if [ -f \"/var/tmp/modified-release\" ]; then rm \"/var/tmp/modified-release\""))
```

Comment:
REQUIRED: this code won't work you at least don't have 'fi' at the end;
And I'm under impression that rm -f would suit much better in this case.
If you persist you can use something like the following
[[ -f blabla ]] && rm blabla